### PR TITLE
add command to verify that a cluster is updated

### PR DESCRIPTION
**Pull request by xh3b4sd**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

## Context 

Based on https://github.com/giantswarm/awscnfm/pull/208. This is the structure and logic for `awscnfm action verify cluster updated`.

### CODE REVIEW

File: cmd/action/verify/cluster/updated/runner.go [hunk 1]
```diff
@@ -1,25 +1,48 @@
-package ac004
+package updated
 
 import (
 	"context"
 
-	"github.com/giantswarm/apiextensions/v2/pkg/apis/infrastructure/v1alpha2"
+	"github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
```
Comment (calvix) [inline cmd/action/verify/cluster/updated/runner.go:6]: is this change intentional?

Comment (xh3b4sd) [reply cmd/action/verify/cluster/updated/runner.go]: It was not. Fixed it. 🤗

**Review by calvix — APPROVED** [review]
